### PR TITLE
SOV-106: Graph configuration

### PR DESCRIPTION
**Pull request by tiltom**
Resolves https://sovryn.atlassian.net/browse/SOV-106

### CODE REVIEW

**Review by soulBit — APPROVED** [review]

File: .changeset/hip-bags-marry.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'frontend': minor
```
Comment (creed-victor) [inline .changeset/hip-bags-marry.md]: why not patch?

```suggestion
'frontend': patch
```

File: package.json [hunk 2]
```diff
@@ -17,7 +17,9 @@
     "coverage": "turbo run coverage",
     "storybook": "turbo run storybook",
     "prepare": "husky install",
-    "release-packages": "turbo run build lint test --filter=./packages/* && changeset publish"
+    "release-packages": "turbo run build lint test --filter=./packages/* && changeset publish",
+    "generate:graphql:fetch:testnet": "turbo run generate:graphql:fetch:testnet",
+    "generate:graphql:fetch:mainnet": "turbo run generate:graphql:fetch:mainnet"
```
Comment (creed-victor) [inline package.json:22]: why do you need these in main package.json too? Is it helper for developer only or also used by CI when deploying?

File: .changeset/hip-bags-marry.md (hunk 1 above)
Comment (soulBit) [reply .changeset/hip-bags-marry.md]: updated this

File: package.json (hunk 2 above)
Comment (tiltom) [reply package.json:22]: It's just for developers for now and I added it to the main `package.json` so I can call it from there. Do you want me to remove it and call it directly on the frontend dapp only?

**Review by creed-victor — APPROVED** [review]

**Pull request merged**